specs/_features/eip7805/fork-choice.md

```diff
@@ -85,35 +164,111 @@ def get_forkchoice_store(anchor_state: BeaconState, anchor_block: BeaconBlock) -
         block_states={anchor_root: copy(anchor_state)},
         checkpoint_states={justified_checkpoint: copy(anchor_state)},
         unrealized_justifications={anchor_root: justified_checkpoint},
-        # [New in EIP-7805]
-        unsatisfied_inclusion_list_blocks=set(),
+        unsatisfied_inclusion_list_blocks=set(),  # [New in EIP7805]
     )
 ```
 
+### New `get_inclusion_list_store`
+
+```python
+def get_inclusion_list_store() -> InclusionListStore:
+    # `cached_or_new_inclusion_list_store` is implementation and context dependent.
+    # It returns the cached `InclusionListStore`; if none exists,
+    # it initializes a new instance, caches it and returns it.
+    inclusion_list_store = cached_or_new_inclusion_list_store()
+
+    return inclusion_list_store
+```
+
+### New `store_inclusion_list`
+
+```python
+def store_inclusion_list(
+    store: InclusionListStore, inclusion_list: InclusionList, is_before_view_freeze_deadline: bool
+) -> None:
+    key = (inclusion_list.slot, inclusion_list.inclusion_list_committee_root)
+
+    inclusion_lists = store.inclusion_lists.setdefault(key, set())
+    equivocators = store.equivocators.setdefault(key, set())
```

Comment:
The `setdefault` stuff here is ugly 😅 I'll look into fixing that in a follow up PR.

Reply:
It's not about `store`. It's just about Python's Dictionary.

I also don't like this part and that's why I asked @jtraglia if we can use DefaultDict. Let me think about how to improve this.